index.js

```diff
@@ -59,21 +60,39 @@ async function getCoreIndex () {
   }
 }
 
-function getVulnerabilityList (currentVersion, data) {
+function getVulnerabilityList (currentVersion, data, systemEnvironment) {
   const list = []
   for (const key in data) {
     const vuln = data[key]
     if (
-      satisfies(currentVersion, vuln.vulnerable) &&
-      !satisfies(currentVersion, vuln.patched)
+      (
+        satisfies(currentVersion, vuln.vulnerable) &&
+        !satisfies(currentVersion, vuln.patched)
+      ) && (
+        (!systemEnvironment || !Array.isArray(vuln.affectedEnvironments)) ||
+        vuln.affectedEnvironments.includes(systemEnvironment) ||
+        vuln.affectedEnvironments.includes('all')
+      )
     ) {
       list.push(`${bold(vuln.cve)}: ${vuln.overview}\n${bold('Patched versions')}: ${vuln.patched}`)
     }
   }
   return list
 }
 
-async function main (currentVersion) {
+const getSystemEnvironment = (platform) => {
+  switch (platform) {
```

Comment:
this list needs to be updated now

Reply:
This was removed as we use now os.platform, see: https://github.com/RafaelGSS/is-my-node-vulnerable/pull/12/commits/2a277aef3d3c19e661433cb2e7def4eb9b636a7c